docs/content/configuration/index.md

```diff
@@ -257,6 +257,14 @@ This config is used to find the [Indexing Service](../design/indexing-service.ht
 |--------|-----------|-------|
 |`druid.selectors.indexing.serviceName`|The druid.service name of the indexing service Overlord node. To start the Overlord with a different name, set it with this property. |druid/overlord|
 
+### Broker Discovery
+
+This config is used to find the [Brokers](../design/broker.html) using Curator service discovery.
+
+|Property|Description|Default|
+|--------|-----------|-------|
+|`druid.selectors.broker.serviceName`|The druid.service name of the broker node. |druid/broker|
```

Comment:
1) choosing default being druid/broker is inline with druid.selectors.indexing.serviceName default druid/overlord , I will change example config files to set it to "broker"
2) will change property name to `druid.selectors.query.serviceName`
3) yes, but for the use case of ui reaching brokers via coordinator for metadata stuff, users should be able to set a service name that is published by a subset of all brokers.